devtools/buildVisualStories.js

```diff
@@ -39,7 +39,9 @@ export default {
 };
 
 export const ${componentName} = () => {
-    let storyNames = Object.keys(stories).filter(story => story !== 'default');
+    let storyNames = Object.keys(stories).filter(story => {
+        return story !== 'default' && story !== 'dev';
```

Comment:
Good idea! Just in case other reviewers are wondering why there's so many visual test changes - this is why. I think it's a good idea to remove the dev stories from the visuals.